apps/studio/prisma/generated/generatedTypes.ts

```diff
@@ -13,7 +13,6 @@ export type Blob = {
 }
 export type Footer = {
   id: string
-  name: string
```

Comment:
why are we dropping this ah?

Reply:
this is an extra prop. i referenced the footer props in config: 

```ts

{
  LinkComponent?: any
  isGovernment?: boolean
  siteName: string
  agencyName: string
  lastUpdated: string
}
```